modules/boss/src/main/ts/ephox/boss/mutant/Properties.ts

```diff
@@ -19,15 +26,19 @@ const document = (_item: Gene): undefined => {
 };
 
 const isText = (item: Gene): boolean => {
-  return item.name === 'TEXT_GENE';
+  return item.name === GeneTypes.Text;
 };
 
 const isComment = (item: Gene): boolean => {
-  return item.name === 'COMMENT_GENE';
+  return item.name === GeneTypes.Comment;
 };
 
 const isElement = (item: Gene): boolean => {
-  return item.name !== undefined && item.name !== 'TEXT_GENE' && item.name !== 'COMMENT_GENE';
+  return item.name !== undefined && item.name !== GeneTypes.Text && item.name !== GeneTypes.Comment;
```

Comment:
```suggestion
  return item.name !== undefined && !isText(item) && !isComment(item);
```

Should this also include `Special`?

Reply:
You could also use `!Type.isUndefined(item.name)` if you are changing this?